src/components/SlideOver/SlideOver.tsx

```diff
@@ -0,0 +1,168 @@
+    </div>
+  )
+
+  return (
+    <React.Fragment>
+      {/* Background overlay */}
+      <Transition
+        show={visible}
+        enter="ease-out duration-200"
+        enterFrom="opacity-0"
+        enterTo="opacity-100"
+        leave="ease-in duration-200"
+        leaveFrom="opacity-100"
+        leaveTo="opacity-0"
+      >
+        <div className="sbui-slideover-overlay-container">
+          <div className="sbui-slideover-overlay"></div>
+        </div>
+      </Transition>
+
+      {/* SlideOver element */}
+      <div className="sbui-slideover-fixed" onClick={onCancel}>
+        <div className="sbui-slideover-absolute">
+          <section className={'sbui-slideover-container ' + orientationClasses}>
+            <Transition
+              show={visible}
+              enter="transform transition ease-in-out duration-500 sm:duration-700"
+              enterFrom={left ? '-translate-x-full' : 'translate-x-full'}
+              enterTo="translate-x-0"
+              leave="transform transition ease-in-out duration-500 sm:duration-700"
+              leaveFrom="translate-x-0"
+              leaveTo={left ? '-translate-x-full' : 'translate-x-full'}
+            >
+              <div className={'w-screen ' + (wide ? 'max-w-2xl' : 'max-w-md')}>
+                <div
+                  className={'sbui-slideover ' + className}
+                  onClick={stopPropagation}
+                >
+                  <Space
+                    size={6}
+                    direction="vertical"
+                    style={{
+                      minHeight: '0',
+                      flex: '1 1 0%',
+                      overflowY: 'scroll',
+                    }}
+                  >
+                    <header className="sbui-slideover-header">
+                      <Space
+                        size={3}
+                        direction="row"
+                        style={{
+                          alignItems: 'center',
+                          justifyContent: 'space-between',
+                        }}
+                      >
+                        {title && (
+                          <Typography.Title className="m-0" level={4}>
+                            {title}
+                          </Typography.Title>
+                        )}
+                        <div className="sbui-slideover-close-container">
+                          <Button
+                            aria-label="Close panel"
+                            onClick={onCancel}
+                            type="text"
+                            shadow={false}
+                            icon={
+                              <Icon size="xlarge" type="X" strokeWidth={2} />
+                            }
+                          />
+                        </div>
+                      </Space>
+                      <div>
+                        {description && (
+                          <Typography.Text>{description}</Typography.Text>
```

Comment:
Not sure about typography for description. Seems it should has a little more secondary look. cc @MildTomato

Reply:
maybe use `<Typography.Text type="secondary">` for the description.

Reply:
Sure, I'll try that out!